examples/src/main/scala/org/apache/spark/examples/snappydata/WorkingWithJson.scala

```diff
@@ -102,6 +102,38 @@ object WorkingWithJson extends SnappySQLJob {
 
     val allPersons = nameAndAddress.toJSON
     allPersons.show(truncate = false)
+
+    // Multi-line json example
+    println("Multi-line json example\n################")
+    val locomotives_path = s"${jobConfig.getString("json_resource_folder")}/locomotives.json"
```

Comment:
Is locomotives.json file already available in quickstart/src/main/resources? If not need to add it there.

Reply:
Noted.